### ISSUE TITLE
Commandbox install hangs

### ISSUE THREAD

**Issue by danielosan** [cfsimplicity/lucee-spreadsheet#131]
Installing package [github:cfsimplicity/lucee-spreadsheet]
Cloning Git URL [https://github.com/cfsimplicity/lucee-spreadsheet.git]
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/C:/Users/ananda/.CommandBox/lib/runwar-3.6.1-SNAPSHOT.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/C:/Users/ananda/.CommandBox/lib/slf4j-nop-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
....[here it hangs]

**Comment by cfsimplicity** [comment]
Sorry, I'm not really au fait with the CommandBox integration which was added by @murphydan Perhaps he can help or someone from Ortus?

**Comment by murphydan** [comment]
I was just able to do it without a problem.

![1__java_and_box_json_ _scs-outlet](https://user-images.githubusercontent.com/5429291/30218090-5bee442e-9475-11e7-91c0-97abafd20ac4.png)

My guess is that this has more to do with the CommandBox setup on your computer in general or permissions and nothing to do with lucee-spreadsheet. This is just trying to install straight from the github repo and not using Forgebox or anything. Can you install anything from a github repo using CommandBox?

@danielosan - are you on the CFML slack (https://cfml.slack.com)? There is a box-products channel that you could ask on as well.

**Comment by danielosan** [comment]
Hi thanks for quick reply.  Not sure what is the cause of the problem. I am using commanbox daily (3.7.0 on Win10-64) and hardly have problems. Maybe some conflict with lib in classpath. Had this before with OSGi bundles.

Since I had to ntegrate it quickly I downloaded it and installed it manually into the coldbox app. Works well. Thanks for the great work!! Because using the Lucee extension is no option for us.
